incubator/hnc/internal/validators/namespace.go

```diff
@@ -151,7 +151,7 @@ func (v *Namespace) illegalCascadingDeletion(ns *forest.Namespace) admission.Res
 		}
 	}
 	if len(cantDelete) != 0 {
-		msg := fmt.Sprintf("Please set allowCascadingDeletion first either in the parent namespace or in all the subnamespaces.\n  Subnamespace(s) without allowCascadingDelete set: %s.", cantDelete)
+		msg := fmt.Sprintf("Please set allowCascadingDeletion in all the subnamespaces.\n  Subnamespace(s) without allowCascadingDelete set: %s.", cantDelete)
```

Comment:
Shouldn't the msg be `Please set ACD in this parent namespace`?

Reply:
This is where the parent has ACD set and the child doesn't

Reply:
Sry I mis-understood the logic, updated

Reply:
(this change also depends on this discussion https://github.com/kubernetes-sigs/multi-tenancy/pull/1257#discussion_r518846183)